docs/7___appendix.adoc

```diff
@@ -32,10 +32,10 @@ A <<Clock>> tick causes a <<Clock>> activation at that time instant, except for
 |See <<time-instant>>.
 
 |_co-simulation_
-|Coupling of several _simulation programs_ in order to compute the global behavior of a system that consists of several subsystems.
+|Coupling of several _simulators_ in order to compute the global behavior of a system that consists of several subsystems.
```

Comment:
Here we we could lose the distinction between "co-simulation" (as a more general concept, could by the co-simulation of multiple stand-alone FMUs) and "simulator coupling" / "tool coupling"  (requiring the installation of different simulation tools and the co-simulation of "tool coupling FMUs)